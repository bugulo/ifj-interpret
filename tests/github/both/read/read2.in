0
-50
1234567890
