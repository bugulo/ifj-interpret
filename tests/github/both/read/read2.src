.IPPcode21

DEFVAR GF@a

# 0
READ GF@a int
WRITE GF@a
TYPE GF@a GF@a
WRITE GF@a

# -50
READ GF@a int
WRITE GF@a
TYPE GF@a GF@a
WRITE GF@a

# 1234567890
READ GF@a int
WRITE GF@a
TYPE GF@a GF@a
WRITE GF@a
