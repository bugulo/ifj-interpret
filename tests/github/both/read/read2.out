0int-50int1234567890int